packages/editor/src/client/api/index.ts

```diff
@@ -20,6 +20,7 @@ export type Scalars = {
   Color: string;
   RichText: Node[];
   Slug: string;
+  value: any;
```

Comment:
Something doesn't seem right here I think, this probably should be uppercase, no?
Something went wrong with generation?

Reply:
Good catch